### ISSUE TITLE
Possible optimization: id column can be removed, link_id column can be made primary key.

### ISSUE THREAD

**Issue by ivansglazunov** [deep-foundation/deeplinks#23]
Requirement: only a single custom table row can be used for any link.
Possible optimization: id column can be removed, link_id column can be made primary key.

**Comment by Konard** [comment]
@ivansglazunov are still you sure we can apply this optimization? As I remember you had some arguments against this optimization.

**Comment by ivansglazunov** [comment]
> @ivansglazunov are still you sure we can apply this optimization? As I remember you had some arguments against this optimization.

Yes. But apart from the pleasure of perfectionism, I do not see any advantages now. This can be done later, or for fun)

**Comment by Konard** [comment]
> > @ivansglazunov are still you sure we can apply this optimization? As I remember you had some arguments against this optimization.
> 
> Yes. But apart from the pleasure of perfectionism, I do not see any advantages now. This can be done later, or for fun)

Well, it will affect memory usage and performance tiny bit as well ;)
